Feature/add knife swep

snepderg opened the pull request:
I created a new PAC SWEP using the content from ~~CW Melee's 'Extrema Ratio Official' knife.~~ CS:S's knife.

Should work about as well as the other PAC SWEPs, at least until we get around to making them better.

snepderg committed:
Add experimental knife PAC SWEP

snepderg committed:
Move 'AddCSLuaFile( "shared.lua" ) to first line

There was a newline at the beginning of the file and I don't know why.

snepderg committed:
Remove Swep Init function from init.lua

This should go in shared.lua I think.

snepderg committed:
Add SWEP init function to shared.lua

This was cut from init.lua

plally commented on lua/weapons/pac_cw_extrema_ratio/cl_init.lua:
```diff
@@ -0,0 +1,6 @@
+include( "shared.lua" )
+
+SWEP.PrintName            = "PAC_CW_EXTREMA_RATIO"
```
either line them up, or put the `=` one space from the variable.

snepderg replied:
I agree, just gonna be a pain to refactor the entire repository. I'll look into it for this file however since Phatso wants me to change it to not be Extrema. (We're gonna try for the CS:S Knife if possible)

plally commented on lua/weapons/pac_cw_extrema_ratio/shared.lua:
```diff
@@ -0,0 +1,55 @@
+SWEP.DrawCrosshair        = false
+SWEP.ViewModelFOV        = 76
+SWEP.ViewModelFlip        = false
+
+SWEP.Slot                = 3
+SWEP.SlotPos            = 1
+SWEP.IconLetter            = "i"
+SWEP.DrawWeaponInfoBox  = true
+SWEP.Category           = "PAC SWEPS"
+
+------------
+-- Info --
+------------
+SWEP.PrintName        = "PAC_CW_EXTREMA_RATIO"
+SWEP.Purpose        = "Imitate CW's Extrema Knife"
+
+
+-------------
+-- Misc. --
+-------------
+SWEP.Spawnable            = true
+SWEP.AdminSpawnable        = true
+
+
+----------------------
+-- Primary Fire --
+----------------------
+SWEP.Primary.NumShots        = 0
+SWEP.Primary.ClipSize        = -1
+SWEP.Primary.DefaultClip    = -1
+SWEP.Primary.Ammo            = "none"
+SWEP.FiresUnderwater         = false
+
+
+-------------------------
+-- Secondary Fire --
+-------------------------
+SWEP.Secondary.ClipSize        = -1
+SWEP.Secondary.DefaultClip    = -1
+SWEP.Secondary.Ammo            = "none"
```
same  comment as before

snepderg committed:
Change CW_Extrema_Ratio to Knife, Fix styling

snepderg commented:
Alright, I changed it to be the CS:S Knife (It isn't packaged so it relies on CS:S mounted on the server), the View/World Models look good, I fixed the styling in regards to spaces (Will be fixed for the rest of the repository but more than likely it's going to get refactored so just ignore it for now, it'll be a different PR in the future)
